spec/support/fixtures/pdf-text-all-yes-answers.txt

```diff
@@ -83,53 +83,52 @@ Intimidator/bully                 Yes
 Sex offender                      Yes
   Adult male                      Yes
   Adult female                    Yes
-W1234BY: McTest Testy                                                         Page 2 of 3
```

Comment:
💡 🔑  if you want to review these pdf changes ignoring the whitespace you can whack `?w=1` to the end of the url. [por exemplo.](https://github.com/ministryofjustice/moving-people-safely/pull/272/files?w=1)

<img width="1878" alt="screen shot 2017-04-21 at 15 33 15" src="https://cloud.githubusercontent.com/assets/1006365/25282272/e476e6fc-26a7-11e7-997d-28c6e5f52716.png">